jsconfig.json

```diff
@@ -3,6 +3,7 @@
         "checkJs": true,
         "declaration": true,
         "declarationMap": true,
+        "declarationDir": "./types",
```

Comment:
One thing we must assure before merging this change is that the extremely commonly used `Option` type is still available without importing in all packages that reference this package in their `tsconfig.json` file.

Reply:
I didn't necessarily intend to include this line - I had some trouble accessing the updated type in any meaningful way from one of the other repos (=~ "not a generic type")

...so I was fiddling with these options to see if I could make a difference (not yet)

Reply:
removed